### DISCUSSION TITLE
Support for glsp server version 2.1.0?

### DISCUSSION THREAD

**Discussion by RobMoloney** [eclipse-emfcloud/emfcloud#233] [All other]
Will modelserver-glsp-integration be updated to support GLSP server version 2.1.0?

**Comment by ndoschek** [comment]
Hi @RobMoloney,

At the moment, the `modelserver-glsp-integration` will not be updated to support a newer GLSP server version , as the project is currently in maintenance mode and not under active development. However, we are working on a new node-based model management framework. If you are interested, we can provide you with updates on this new framework once it is ready.

Cheers,
Nina
